### PR TITLE
Manual to translate Flux to Lux

### PR DESCRIPTION
**Pull request by avik-pal**

### CODE REVIEW

**Comment by avik-pal** [comment]
@ToucheSir what is the correct way of getting gradients for only trainable parameters in Flux? I know the params way http://lux.csail.mit.edu/previews/PR69/manual/migrate_from_flux/#implementing-custom-layers but want to avoid it.

File: docs/src/manual/migrate_from_flux.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    end
+
+
+
+
+
+
+
+    # `A` is not trainable
+    Flux.trainable(f::FluxLinear) = (f.B,)
```
Comment (ToucheSir) [inline docs/src/manual/migrate_from_flux.md:105]: What you wrote is the way to go (default to `@functor [Layer] [fields...]` unless you have non-trainable state, in which case `trainable` is necessary as well (BTW this is exported by Optimisers now, so you can define layers without importing Flux at all. Perhaps that should be the recommendation).

The only change to make is this:
```suggestion
    Flux.trainable(f::FluxLinear) = (; B=f.B)
```
Not including prop names in `trainable` is deprecated and will be removed in the next major version of Optimisers.

Comment (ToucheSir) [reply docs/src/manual/migrate_from_flux.md:105]: (1 hour during a work day lunch break is a little short notice for reviewing a PR 😂 )

Comment (avik-pal) [reply docs/src/manual/migrate_from_flux.md:105]: I needed the manual to go through for a corresponding DiffEqSensitivity PR. I will update the example in a new PR.

**Pull request merged**